Jenkins/update_ci_win_docker_image.ps1

```diff
@@ -0,0 +1,83 @@
+
+$ErrorActionPreference = "Stop"
+
+$globalVariables = (Resolve-Path "$PSScriptRoot\..\global-variables.ps1").Path
+$ciUtils = (Resolve-Path "$PSScriptRoot\..\Modules\CIUtils").Path
+
+Import-Module $ciUtils
+. $globalVariables
+
+$WORKSPACE="$env:WORKSPACE"
+$DOCKER_HUB_USER="$env:DOCKER_HUB_USER"
+$DOCKER_HUB_USER_PASSWORD="$env:DOCKER_HUB_USER_PASSWORD"
+$DOCKERFILE_PATH="$PSScriptRoot\ci-image-Dockerfile"
+$IMAGE_FILES_DIR="$WORKSPACE\files"
+$IMAGE_FILES_BASE_URL="$LOG_SERVER_BASE_URL/downloads"
+
+if (!$DOCKER_HUB_USER) {
```

Comment:
Move the checks for `$env:DOCKER_HUB_USER` and `$env:DOCKER_HUB_USER_PASSWORD` at the beginning of the script, just before you do the imports.